### PR TITLE
Documentation for walltime configuration

### PR DESCRIPTION
**Pull request by fviale**

### CODE REVIEW

**Review by andrewssobral — APPROVED** [review]

**Review by mcompastie — APPROVED** [review]

File: src/docs/user/ProActiveUserGuide.adoc [hunk 1]
```diff
@@ -1561,14 +1561,50 @@ If automatic restarts does not fix errors, the Scheduler applies one of the foll
...
-The walltime mechanism is started
-just before a task is launched. If a task does finish before its walltime, the mechanism is canceled. Otherwise,
-the task is terminated. Note that the tasks are terminated without any prior notice.
+The walltime mechanism is started just before a task is launched.
+If a task does finish before its walltime, the mechanism is canceled. Otherwise, the task is terminated.
+Note that the tasks are terminated without any prior notice.
+
+The walltime can also be configured using the `WALLTIME` <<_generic_information>>. The syntax of this generic information is similar to the task `walltime` attribute.
+The interest of using a generic information rather than a task attribute is that the `WALLTIME` generic information, if defined at job-level, can be applied to all tasks of a waorkflow.
+
```
Comment (AlexFalessi) [inline src/docs/user/ProActiveUserGuide.adoc:1586]: Small typo "waorkflow"

File: src/docs/user/ProActiveUserGuide.adoc [hunk 2]
```diff
@@ -1561,14 +1561,50 @@ If automatic restarts does not fix errors, the Scheduler applies one of the foll
...
 
-The walltime mechanism is started
-just before a task is launched. If a task does finish before its walltime, the mechanism is canceled. Otherwise,
-the task is terminated. Note that the tasks are terminated without any prior notice.
+The walltime mechanism is started just before a task is launched.
+If a task does finish before its walltime, the mechanism is canceled. Otherwise, the task is terminated.
+Note that the tasks are terminated without any prior notice.
+
+The walltime can also be configured using the `WALLTIME` <<_generic_information>>. The syntax of this generic information is similar to the task `walltime` attribute.
+The interest of using a generic information rather than a task attribute is that the `WALLTIME` generic information, if defined at job-level, can be applied to all tasks of a waorkflow.
```
Comment (mboussaa) [inline src/docs/user/ProActiveUserGuide.adoc:1585]: waorkflow => workflow

**Comment by mboussaa** [comment]
@fviale what happens to the job/task state when the timeout is reached? is it considered as a faulty task/job? I guess no, so it is worth mentioning it in the doc

**Comment by fviale** [comment]
@mboussaa yes it is considered as a faulty task. Walltime only apply to tasks.

**Pull request merged**